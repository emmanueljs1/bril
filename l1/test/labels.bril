@main {
  one: int = const 1;
.hello:
  print one;
.world:
  print one;
}
