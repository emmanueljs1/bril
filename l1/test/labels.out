entry: 0
exit: 1
blocks:
block: 0
{"label":"start"}
block: 1
{"label":"end"}
edges:
0 -> 1
